htdocs/themes/berlin_mcjc/sass/_screen-custom.scss

```diff
@@ -7,9 +7,9 @@
 /* @group ----- Generic ----- */
```

Comment:
@arevalok91 is this just formatting changes? If so, could you remove the file from the commit?